circle.yml

```diff
@@ -52,7 +52,7 @@ test:
             > $CIRCLE_ARTIFACTS/server_mem_fog.txt
         & bash wait_for_local_port.bash 8000 40
         && cd tests/functional/fog && rspec tests.rb
-    - S3BACKEND=mem npm start
+    - S3BACKEND=mem MPU_TESTING=yes npm start
```

Comment:
minor comment is that since we use `AWS_ON_AIR=true` we could also use `true` instead of `yes` here to be more consistent. But it's not something I consider crucial.